cmd/sabayon/main.go

```diff
@@ -21,6 +21,10 @@ func main() {
 	flag.Parse()
 
 	var domains = strings.Split(os.Getenv("ACME_DOMAIN"), ",")
+	// trim potential whitespace
+	for _, domain := range domains {
+		domain = strings.TrimSpace(domain)
```

Comment:
It's ok. You're the second person to write their first line of Go with this project. That's awesome :)